README.md

```diff
@@ -261,6 +262,16 @@ mvn clean install -am -Dgib.argsForNotImpactedModules='-Denforcer.skip -Dcheckst
 
 Can be combined with `gib.skipTestsForNotImpactedModules`.
 
+### gib.buildAnway
+
+Defines artifact ids of modules to build anyway, even if these modules have not been changed and/or do not depend on changed modules. Example:
+```
+mvn clean install -Dgib.buildAnyway='unchanged-module-1 unchanged-module-2'
```

Comment:
Comma separated list would be more similar to common mvn approach e.g. [-pl ](https://books.sonatype.com/mvnref-book/reference/_using_advanced_reactor_options.html#_specifying_a_subset_of_projects).

projectsToBuildAnyway, buildAnywayProjects or alwaysBuildProjects may be better name indicating also that the property specifies list of projects.

Reply:
or forceBuildProjects

Reply:
Hm, regarding the comma separation: `argsForNotImpactedModules` also uses spaces, so I wanted to keep that consistent. Not sure. I'm fine with both, your choice! 😁 

I also wanted to align the name with `buildAll`and I wanted to keep it short.
If we are to change the name I'd suggest `forceBuildModules` (which is consistent with the other `*Modules*` properties).

WDYT?

Reply:
I think argsForNotImpactedModules couldnt use anything else but spaces without some additional escaping. I would go with commas here.

forceBuildModules sounds better to me.

These are just mostly nitpicks so it is up to you.

Reply:
Ok, I switched to `forceBuildModules` and comma separation. I also added another test to verify that multiple modules are evaluated as expected. Going to merge right away...